any_parser/any_parser.py

```diff
@@ -221,6 +283,8 @@ def async_extract(
             process_type = ProcessType.FILE
         elif model == ModelType.PRO:
             process_type = ProcessType.FILE_REFINED_QUICK
+        elif model == ModelType.PARSE_WITH_LAYOUT:
+            process_type = ProcessType.PARSE_WITH_LAYOUT
```

Comment:
let's not nest the all different `parse` into the same async_extract. Let's add a new SDK called `async_parse_with_layout`.

Reply:
ack